fetch.py

```diff
@@ -27,13 +27,13 @@
 import argparse
 import sys
 
-from lib.fetch_players import fetch_players
-from lib.fetch_teams import fetch_teams
-from lib.fetch_player_games import fetch_player_games
-from lib.fetch_player_stats import fetch_player_stats
-from lib.fetch_player_boxscores_by_game import fetch_player_boxscores_by_game
+from lib.player.all import fetch_players
+from lib.team.all import fetch_teams
+from lib.player.games_by_season import fetch_player_games
+from lib.player.career_stats import fetch_player_stats
+from lib.game.boxscore import fetch_player_boxscores_by_game
```

Comment:
For now, we can keep using the old modules in this fetch.py function. Once the rest of the refactor is done we can update the references in this file